portal/apps/org.wso2.carbon.iam.user-portal.app/src/main/modules/recovery-manager.js

```diff
@@ -62,7 +62,8 @@ var recoveryManager = {};
     }
 
     /**
-     *  Check whether password recovery enabled.
+     * Check whether the password recovery enabled
```

Comment:
The javadoc comments need to end with a dot. Otherwise, build fails in windows environment because of checkstyle.